go-controller/pkg/ovn/controller/services/svc_template_var.go

```diff
@@ -215,3 +210,72 @@ func getTemplatesFromRulesTargets(rules []LBRule) TemplateMap {
 	}
 	return templates
 }
+
+// NodeIPTemplates maintains templates variables for many IP addresses per node,
+// creating template variables in the for NODEIP_IPv4_0, NODEIP_IPv4_1, NODEIP_IPv4_2, ...
+// if and when they are needed.
+type NodeIPsTemplates struct {
+	ipFamily           corev1.IPFamily
+	templateValues     []*chassisTemplateValue
+	chassisIDtoIPCount map[string]int
```

Comment:
Is this tracking the number of nodeIPs per chassis?

Reply:
yes, going to rename it